vignettes/translation-function.Rmd

```diff
@@ -16,25 +16,26 @@ options(tibble.print_min = 6L, tibble.print_max = 6L, digits = 3)
 There are two parts to dbplyr SQL translation: translating dplyr verbs, and translating expressions within those verbs. This vignette describes how individual expressions (function calls) are translated; `vignette("translation-verb")` describes how entire verbs are translated.
 
 ```{r}
-#| message = FALSE
 library(dbplyr)
-library(dplyr)
+library(dplyr, warn.conflicts = FALSE)
 
-con <- simulate_dbi()
+lf <- lazy_frame(x = 1, y = 2, g = "a")
 ```
 
-`dbplyr::translate_sql()` powers translation of individual function calls, and I'll use it extensively in this vignette to show what's happening. You shouldn't need to use it ordinary code as dbplyr takes care of the translation automatically.
+In this vignette, I'll use `lazy_frame()` to create a toy lazy table that allows us to see the translation without needing to connect to a real database:
```

Comment:
nit: It looks like we use "I'll use" after it's already used, maybe put the `lf <- ...` in the next cell?